services/server/package.json

```diff
@@ -15,7 +15,7 @@
     "start": "node ./dist/server/cli.js",
     "postgres-test:start": "docker-compose -f test/docker-compose.yml up -d",
     "postgres-test:stop": "docker-compose -f test/docker-compose.yml rm -fsv",
-    "postgres-test:migrate": "cd ../database && git submodule update --init && export DATABASE_URL=postgres://sourcify:sourcify@127.0.0.1:${DOCKER_HOST_POSTGRES_TEST_PORT}/sourcify?sslmode=disable && export DBMATE_SCHEMA_FILE=./sourcify-database.sql && npm run migrate:load-schema",
+    "postgres-test:migrate": "cd ../database && git submodule update --init && export DATABASE_URL=postgres://sourcify:sourcify@127.0.0.1:${DOCKER_HOST_POSTGRES_TEST_PORT}/sourcify?sslmode=disable && export DBMATE_SCHEMA_FILE=./sourcify-database.sql && npm run migrate:up",
```

Comment:
Why this change?

Reply:
loading the schema via running the migrations prevents running into this.